packages/web/app/tailwind.config.ts

```diff
@@ -312,7 +312,7 @@ function asd(): PluginCreator {
 }
 
 function addVariablesForColors({ addBase, theme }: PluginAPI) {
-  const allColors: Record<string, string> = flattenColorPalette(theme('colors'));
```

Comment:
I used declaration merging to define the type for `flattenColorPalette`, so we don't need an annotation here.